### ISSUE TITLE
Proxying to localhost no longer works

### ISSUE THREAD

**Issue by macinta** [FelisCatus/SwitchyOmega#1762]
Hello,

Proxy requests to localhost have stopped working for me.  I have a profile which uses a SOCKS4 proxy for all HTTP requests, **including** requests to localhost - i.e., I have an empty bypass list.  This had been working for several months, but it no longer does.  Now HTTP requests to localhost are not sent through the proxy.  I think the last time I used this profile was about 3 weeks ago, so I guess a Chrome update since then broke things.

Thanks,
- Tim Macinta

SwitchyOmega 2.5.20
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36

**Comment by benze** [comment]
I have the same issue; any endpoint that is 127.0.0.1 fails no matter the name in the URL.  I've tried localhost, a machine name, a fake name in my hosts file that resolves to 127.0.0.1 and in all cases, the proxy setting is ignored altogether.

I have only started seeing this behaviour following an upgrade to Chrome 72.0.3626.121
Running on Win 10

Thanks,

Eric

**Comment by ViktorijaAdo** [comment]
Also, having the same issue. But at least it works if I use my machine IP.

Would be nice to have this fixed.

Thanks,
Viktorija

**Comment by mfigueroa** [comment]
Having this issue as well.

**Comment by yyysuo** [comment]
got the same problem

**Comment by AndreGeng** [comment]
+1

**Comment by robxu9** [comment]
I was looking for a solution for this as well and I saw this: https://www.reddit.com/r/chrome/comments/as84m3/chrome_doesnt_give_a_damn_about_proxy_on/eierebt/

**Comment by yyysuo** [comment]
> I was looking for a solution for this as well and I saw this: https://www.reddit.com/r/chrome/comments/as84m3/chrome_doesnt_give_a_damn_about_proxy_on/eierebt/
> 
> tl;dr add `<-loopback>` to Proxy SwitchyOmega's bypass list.

Like this？I can still access my hfs web page  127.0.0.1.

127.0.0.1
[::1]
localhost
<-loopback>

**Comment by yyysuo** [comment]
even when i delete 127.0.0.1 in my bypass list and using an Incognito window

**Comment by robxu9** [comment]
No, delete 127.0.01, [::1], and localhost

**Comment by yyysuo** [comment]
> No, delete 127.0.01, [::1], and localhost

I have checked <-loopback> and loopback，only one of them left in my bypass list，and when I access127.0.0.1 using Incognito window，I can still see the hfs web page load successfully。

**Comment by robxu9** [comment]
it's `<-loopback>`, and are you allowing Proxy SwitchyOmega to run in incognito?
I'm out of suggestions otherwise, it's definitely working on my end.

**Comment by yyysuo** [comment]
> it's `<-loopback>`, and are you allowing Proxy SwitchyOmega to run in incognito?
> I'm out of suggestions otherwise, it's definitely working on my end.

Thank you very much for your reply，I think I should check other configurations.

**Comment by teodor440** [comment]
Hey, so a hack to proxy localhost with switchy omega can be to enter a domain that points to localhost (127.0.0.1). One such domain is spoofed.burpcollaborator.net

**Comment by tianjianchnbytedance** [comment]
Or use Chrome 69 in mac with system proxy(or SwitchyOmega if you prefer such extra work)

**Comment by tianjianchn** [comment]
Or use Chrome 69 in mac with system proxy(or SwitchyOmega if you prefer such extra work)

**Comment by oscar6echo** [comment]
On Ubuntu 20.04 LTS I'm running -  a local privoxy in docker: [arch-delugevpn](https://github.com/binhex/arch-delugevpn) which is running a wireguard client, connected to a wireguard server on a VM somewhere in the cloud.

This works fine when I set the system proxy to localhost:8118 with IgnoreHosts=localhost,127.0.0.0/8,::1.  
Then I can browse on chrome and check I am abroad with e.g.
+ [dnsleaktest](https://www.dnsleaktest.com/)
+ [whatismyipaddress.com](https://whatismyipaddress.com/)

But there is *NO* way to input these proxy settings in SwitchyOmega.  
Whenever I visit a website with a proxy defined in SwitchyOmega, I get a ERR_TIMED_OUT error.  
Obviously the request does not go through the local privoxy.  
For some reasons, either SwitchyOmega config is ineffective and Chrome cannot reach the proxy on localhost:8118. 
Note that all works as expected when I use [DIRECT] or [System Proxy].  

I tried all combinations (I think) of:
+ IP: localhost, 127.0.0.1, public IP addr
+ Bypass list: `<-loopback>`, `localhost`, `127.0.0.1`, `::1`

The issue is clearly: Why does my privoxy config (localhost or equivalent:8118) get ignored by Chrome ?

Anybody has already met/solved this problem ?
Any hint appreciated.
